### PR TITLE
[Documentation][Book] Themes

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | no |
| New docs?    | yes |
| Related tickets | partially #5275 |
| License         | MIT |

### CODE REVIEW

File: docs/book/themes/themes.rst [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+1. Inside of the ``app/themes/`` directory create a new directory for your theme:
+
+Let it be ``CrimsonTheme/`` for instance.
+
+2. Create ``composer.json`` for your theme:
+
+.. code-block:: yaml
+
+   {
+       "name": "sylius/crimson-theme",
```
Comment (pjedrzejewski) [inline docs/book/themes/themes.rst]: We should suggest "acme" as vendor name.

File: docs/book/themes/themes.rst [hunk 2]
```diff
@@ -0,0 +1,110 @@
+.. index::
+   single: Themes
+
+Themes
+======
+
+Theming is method of customizing how do your channels look like in Sylius. Each channel can have a different Theme.
```
Comment (venyii) [inline docs/book/themes/themes.rst]: Theming is **a** method [...] how ~do~ your channels [...]

Theme is capitalized at the end, but lowercased in the rest of the text

File: docs/book/themes/themes.rst [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+.. code-block:: yaml
+
+   sylius_theme:
+       sources:
+           filesystem: ~
+
+How to create themes?
+---------------------
+
+Let's see how to customize the view of cart widget inside of your custom theme.
```
Comment (venyii) [inline docs/book/themes/themes.rst]: [...] of **the** cart widget [...]

File: docs/book/themes/themes.rst [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+5. If changes are not yet visible, clear the cache:
+
+.. code-block:: bash
+
+   $ php bin/console cache:clear
+
+.. note::
+
+   You can override any template of Sylius like that, but also add the ``web/assets/`` inside of the theme directory to override the css or js files.
```
Comment (venyii) [inline docs/book/themes/themes.rst]: The second part sounds a bit off, maybe something like:

You can override any template of Sylius like that, as well as static files by adding a ``web/assets/`` directory in the theme directory to override CSS or JS files.

**Comment by TheMadeleine** [comment]
@pjedrzejewski fixed ;)

**Comment by pjedrzejewski** [comment]
Nice, thank you Magda!

File: docs/book/themes/themes.rst [hunk 5]
```diff
@@ -0,0 +1,105 @@
+.. index::
+   single: Themes
+
+Themes
+======
+
+Theming is a method of customizing how your channels look like in Sylius. Each channel can have a different tqheme.
```
Comment (michalmarcinkowski) [inline docs/book/themes/themes.rst]: `tqheme`? 😄

Comment (gabiudrescu) [inline docs/book/themes/themes.rst]: I'd add more information about why using themes. From my understanding:
 - use standard Symfony overriding technique if:
 -- you have only one channel
 -- you don't need multiple themes at the same time configured
 -- you want something basic (e.g. change the color set of the website or re-arrange some  blocks)

 - use Sylius themes overriding technique if:
 -- you have more than one channel enabled for a single Sylius instance and every channel should have their own look and behavior
 -- etc. etc.

Comment (michalmarcinkowski) [inline docs/book/themes/themes.rst]: Well described @gabiudrescu, I agree.

@TheMadeleine it's important to put this information in the docs, there are already questions about that.

Comment (gabiudrescu) [inline docs/book/themes/themes.rst]: I wish someone would have taught me that 6 months ago :grin:

**Pull request merged**